src/components/DataInput/CheckboxGroupInput.tsx

```diff
@@ -0,0 +1,31 @@
+"use client";
+
+import React from "react";
+import { Checkbox, FormControl, FormControlLabel, FormGroup, FormLabel } from "@mui/material";
+
+interface Props {
+    labelsAndKeys: [string, string][];
+    groupLabel?: string;
+    onChange?: (event: React.ChangeEvent<HTMLInputElement>) => void; // eslint-disable-line no-unused-vars
+}
+
+const CheckboxInput: React.FC<Props> = (props) => {
+    return (
+        <FormControl>
+            <FormLabel>{props.groupLabel}</FormLabel>
```

Comment:
if `props.groupLabel` is undefined, should this be label with empty string or no label at all?

Reply:
If `props.groupLabel` is `undefined` then the label just isn't rendered. It's all handled by the MUI component